controllers/testdata/declarative/case_2/expected/created/sample-pipeline.yaml.tmpl

```diff
@@ -29,7 +29,7 @@ data:
               "parent_task": "data-prep"}], "validate-model": [{"name": "train-model-model",
               "parent_task": "train-model"}]}'
             tekton.dev/artifact_bucket: mlpipeline
-            tekton.dev/artifact_endpoint: minio-service.kubeflow:9000
+            tekton.dev/artifact_endpoint: $(MINIO_SERVICE_SERVICE_ENDPOINT)
```

Comment:
I like the intent here, but unfortunately you can see the old value used for `tekton.dev/artifact_endpoint` does not include the scheme (ie http://) as that is provided in the `tekton.dev/artifact_endpoint_scheme` annotation directly below.  That means we cant' use `{{.ObjectStorageConnection.Endpoint}}` and instead need to construct the endpoint via "{{.ObjectStorageConnection.Host}}:{{.ObjectStorageConnection.Port}}"

Reply:
Done, updated to yse "${MINIO_SERVICE_SERVICE_HOST}:${MINIO_SERVICE_SERVICE_PORT}" instead of  `{{.ObjectStorageConnection.Endpoint}}`

Reply:
looks like you might've used regular brackets instead of curly, which would cause problems - $(foo) evaluates `foo` as a command, ${foo} is the value of `foo` as an env var

Reply:
Updated to have curly brackets instead.